### PR TITLE
New version of script

### PR DESCRIPTION
**Pull request by Jonastjep**
This update has the main script which does the whole process of fitting and then creates pdf files and a big dictionary file with the fitting parameters for all data of all years with new additions, as well as a library containing a function that can be used to fit specific files, returning a small library with fitting parameters that can then be used in another context. This update also brings a dictionary file which contains all the default fitting parameters and that will be progressively filled with data specific values in case the default fitting process doesn't give good enough results.

### CODE REVIEW

**Comment by jackfrost373** [comment]
@Jonastjep Is the branch ready to merge?

**Comment by Jonastjep** [comment]
Almost, I just cannot manage to get any results when I replace the the two normalization arguments by only one, the whole fitting becomes very strange and returns nonsensical graphs/fits... Otherwise it's ready for merging

File: analysis/Scripts/MassFitting/MainProgram.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+		myCB        = ROOT.RooCBShape("myCB","Crystal Ball", mass, gauss_mean, cb_width, cb_alpha, cb_n)
+		
+		exponential = ROOT.RooRealVar("exponential","C", exponential_range[0], exponential_range[1], exponential_range[2])
+		myexponential = ROOT.RooExponential("myexponential","Exponential", mass, exponential)
+		exponential_Norm  = ROOT.RooRealVar("exponential_Norm","Exponential Yield", mctree.GetEntries()/nbins * 3/exponential_normalisation_factor, 0, mctree.GetEntries() * 2)
+
+		gauss_Norm  = ROOT.RooRealVar("gauss_Norm","Gauss Yield", mctree.GetEntries()/nbins * 3, 0, mctree.GetEntries() * 2)
+		cb_Norm     = ROOT.RooRealVar("cb_Norm","CB Yield", mctree.GetEntries()/nbins * 3/cb_normalisation_factor, 0, mctree.GetEntries() * 2)
+		
+		Actual_signalshape = ROOT.RooAddPdf ("Actual_signalshape", "Shape of the interesting events", ROOT.RooArgList(myGauss, myCB), ROOT.RooArgList(gauss_Norm, cb_Norm))
```
[the remark this answers is not in the document]
Comment (Jonastjep) [reply analysis/Scripts/MassFitting/MainProgram.py]: I tried to solve this but it gives very strange results... I replaced by the following to try out:
```
combined_Norm  = ROOT.RooRealVar("gauss_Norm","Gauss Yield", mctree.GetEntries()/nbins * 3, 0, mctree.GetEntries() * 2)
Actual_signalshape = ROOT.RooAddPdf ("Actual_signalshape", "Shape of the interesting events", ROOT.RooArgList(myGauss, myCB), ROOT.RooArgList(combined_Norm))
```

[the remark this answers is not in the document]
Comment (jackfrost373) [reply analysis/Scripts/MassFitting/MainProgram.py]: The combined norm should be between 0 and 1, as it is a fraction. Initialize it at 0.5 at first.

**Comment by Jonastjep** [comment]
@jackfrost373 Alright! Everything has been fixed!

**Pull request merged**